tool/tctl/common/user_command.go

```diff
@@ -203,13 +203,6 @@ func (u *UserCommand) Add(ctx context.Context, client auth.ClientI) error {
 	u.allowedLogins = flattenSlice(u.allowedLogins)
 	u.allowedWindowsLogins = flattenSlice(u.allowedWindowsLogins)
 
-	// Validate roles (server does not do this yet).
```

Comment:
Perhaps "Remove in XXX" instead? Use case: new tctl, old server.

Reply:
Oh nice one, should have considered that. 
Would the XXX here be a specific version or should I keep it general and just say, "Remove once check added to server has been released"?

Reply:
Specific, latestRelease+2, so right now this is:
```go
// DELETE IN 12.0.0
```